static/templates/subscription_table_body.hbs

```diff
@@ -11,10 +11,9 @@
             <div class="left">
                 <div class="search-container">
                     <form id="add_new_subscription" class="form-inline" action="">
-                        <i id="search_stream_name_icon" class="fa fa-search" aria-hidden="true" data-toggle="tooltip" data-original-title="Filter streams (q)"></i>
+                        <input type="text" name="stream_name" id="search_stream_name_top" placeholder="{{t 'Filter streams' }}" value="" autocomplete="off" />
```

Comment:
Merging the 1st commit and this one would eliminate this unecessary addition and deletion.

Reply:
Done!